### PR TITLE
<FeatureList/> - new component

### PR DESCRIPTION
**Pull request by sivanharel**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://www.wix.com/pages/wix-style-react.pr_5752/

File: src/MarketingPageFeaturesFooter/docs/index.story.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+export default {
+  category: storySettings.category,
+  storyName: storySettings.storyName,
+
+  component: MarketingPageFeaturesFooter,
+  componentPath: '..',
+
+  componentProps: {
+    buttonText: 'Hello World!',
```
Comment (sivanharel) [inline src/MarketingPageFeaturesFooter/docs/index.story.js:34]: Remove it

File: src/MarketingPageFeaturesFooter/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+
+export interface MarketingPageFeaturesFooterProps {
+  dataHook?: string;
+  className?: string;
```
Comment (sivanharel) [inline src/MarketingPageFeaturesFooter/index.d.ts:5]: Remove it

File: src/MarketingPageFeaturesFooter/MarketingPageFeaturesFooter.js [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+          )
+        }
+      />
+    )}
+    <div className={styles.featureItemTextContainer}>
+      {title && (
+        <div className={styles.featureItemTitleContainer}>
+          <Text
+            data-hook={`${dataHooks.featureTitle}${index}`}
+            size="small"
```
Comment (sivanharel) [inline src/MarketingPageFeaturesFooter/MarketingPageFeaturesFooter.js:57]: change to 'medium'

File: src/MarketingPageFeaturesFooter/docs/index.story.js [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+    }),
+
+    tabs([
+      tab({
+        title: 'Description',
+        sections: [
+          description({
+            title: 'Description',
+            text:
+              'This component is a footer for the MarketingPageLayout component.',
```
Comment (sivanharel) [inline src/MarketingPageFeaturesFooter/docs/index.story.js:83]: change to 'This component is used as a footer for the Marketing Page Layout.'

**Comment by sivanharel** [comment]
@milkyfruit - I will make this component generic as well. And if you have more comments please let me know :)

**Comment by milkyfruit** [comment]
@sivanharel yes, I think it can be renamed to `FeatureList`
I noticed this component behaves a bit different than TestimonialList, because it supports only 3 items per row,  then creates a grid:
image

It's ok to support grid as well, but I think both components should have the same logic and then there should be `columns` prop, in case someone wants to show 4 or 2 features per row.

**Comment by sivanharel** [comment]
@milkyfruit - Yes, I implemented two options and waited for your answer to the question I asked in the Zeplin :)
It is up to you, tell me what is the behavior that you prefer and I will implement it in the
 "featureList" as well.
For the grid option, it sounds like a great idea to add a columns prop that will determine the number of columns in the list.

**Review by shlomitc — CHANGES_REQUESTED** [review]

File: src/FeatureList/FeatureList.uni.driver.js [hunk 5]
```diff
@@ -0,0 +1,52 @@
+import { baseUniDriverFactory } from 'wix-ui-test-utils/base-driver';
+import { dataHooks } from './constants';
+import { textUniDriverFactory } from '../Text/Text.uni.driver';
+
+export const featureListDriverFactory = (base, body) => {
+  const byHook = hook => base.$(`[data-hook*="${hook}"]`);
```
Comment (shlomitc) [inline src/FeatureList/FeatureList.uni.driver.js:6]: can this be refactored to some general place? probably we have tons of these

File: src/common/FluidColumns/index.d.ts [hunk 6]
```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+
+export interface FluidColumnsProps {
+  dataHook?: string;
+  className?: string;
+  cols?: number;
+  children: React.ReactNode;
+}
+
+export default class FluidColumns extends React.PureComponent<FluidColumnsProps>{}
```
Comment (shlomitc) [inline src/common/FluidColumns/index.d.ts:10]: not a PureComponent

**Review by shlomitc — APPROVED** [review]
minor changes and we can merge

**Pull request merged**